package.json

```diff
@@ -81,8 +93,8 @@
   ],
   "license": "MIT",
   "scripts": {
-    "start": "node --max-old-space-size=2048 node_modules/react-static/bin/react-static-start",
-    "stage": "react-static build --staging",
+    "start": "node --max-old-space-size=2048 node_modules/react-static/bin/react-static start",
+    "stage": "cross-env REACT_STATIC_STAGING=true react-static build --staging",
```

Comment:
We were relying on an env setting that react-static 7 no longer sets to diferentiate between staging and production.